packages/ra-ui-materialui/src/field/FileField.tsx

```diff
@@ -87,8 +90,6 @@ export const FileField = <
         );
     }
 
-    const titleValue = get(record, title)?.toString() || title;
```

Comment:
you've lost the `toString()` in the process.

Reply:
Added back